MotionPhotoReader/app/src/main/java/com/google/android/libraries/motionphotoreader/MotionPhotoInfo.java

```diff
@@ -26,11 +25,13 @@
  * Contains information relevant to extracting frames in a Motion Photo file.
  *
  * The fields stored are:
- *   - The width of the video (before camera orientation rotations are applied) in pixels.
- *   - The height of the video (before camera orientation rotations are applied) in pixels.
- *   - The duration of the video in microseconds.
- *   - The camera orientation while the motion photo was taken, as a rotation in degrees.
- *   - The byte offset from the end of the file at which the video track begins.
+ *   width: The width of the video (before camera orientation rotations are applied) in pixels.
+ *   height: The height of the video (before camera orientation rotations are applied) in pixels.
+ *   durationUs: The duration of the video in microseconds.
+ *   rotation: The camera orientation while the motion photo was taken, as a rotation about the
+ *   z-axis, measured in degrees.
```

Comment:
My previous comment was about positive vs. negative rotation (ccw vs cw). Could you add that too?